### ISSUE TITLE
Redirects not being returned

### ISSUE THREAD

**Issue by Hudsonzp** [Webbist-dev/strapi-redirects#18]
I was using 0.0.6, and it worked. Upgraded to the latest and also tested 1.0.3, but now I'm having trouble fetching the redirects. Returning null even though there are redirects set in the admin and I restarted. Note - I'm not the best developer so I must be missing something obvious here....

Here's my Redirects.js for the frontend (Next js)

```
const redirects = () => {
  const token = process.env.NEXT_PUBLIC_TOKEN;
  const options = {headers: {Authorization: `Bearer ${token}`}};
  const mergedOptions = {
    next: {revalidate: 60},
    headers: {
      'Content-Type': 'application/json',
    },
    ...options,
  };

return fetch('http://localhost:1337/redirects', mergedOptions)
    .then(res => res.json())
    .then(response => {
      console.log(response.data);
};
```

Next.js config
```
redirects: () => getRedirects(),
```

![CleanShot 2024-04-18 at 09 52 49](https://github.com/Webbist-dev/strapi-redirects/assets/6841693/0bfcd229-cd13-438e-934f-1b9f79d3b7b9)

![CleanShot 2024-04-18 at 09 36 32](https://github.com/Webbist-dev/strapi-redirects/assets/6841693/afa936d4-79c9-460a-86d8-3dc82f5ca82c)

![CleanShot 2024-04-18 at 09 39 57](https://github.com/Webbist-dev/strapi-redirects/assets/6841693/48b31d02-e2f5-4a86-826a-01a98d18b829)

**Comment by Hudsonzp** [comment]
I'm seeing the 401 now. So this is definitely something dumb on my part.

**Comment by Webbist-dev** [comment]
This is an easy one. 
I actually fixed an issue from 0.3 which allowed you access to the redirect admin route at /redirects. This route no longer exists publicly, instead go to api/redirects :)
